src/Sarif.Multitool.FunctionalTests/TestData/DeserializationError.sarif

```diff
@@ -1,10 +1,10 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-01-09",
-  "version": "2.0.0-csd.2.beta.2019-01-09",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-02-20",
+  "version": "2.0.0-csd.2.beta.2019-02-20",
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner"
+        "driver": { "name": "CodeScanner" }
```

Comment:
>driver [](start = 9, length = 6)

These required changes are what prompted us to disable jschema previously. as you recall, i'm sure, the validator 'works' even w/breaking SARIF changes but only as long as it isn't required to return any errors. After updating, I was able to see and fix errors from the test files.